examples/wsclient/main.go

```diff
@@ -1387,6 +1388,34 @@ func callPricesWSStream(w *provider.WSClient) bool {
 	return false
 }
 
+func callGetTickersWSStream(w *provider.WSClient) bool {
+	log.Info("starting ticker stream")
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	stream, err := w.GetTickersStream(ctx, &pb.GetTickersStreamRequest{
+		Project: pb.Project_P_OPENBOOK,
+		Markets: []string{"BONK/SOL", "wSOL/RAY", "BONK/RAY", "RAY/USDC",
```

Comment:
should we make some of these addresses?

Reply:
agree

Reply:
I hope you took these markets from the file of markets we support